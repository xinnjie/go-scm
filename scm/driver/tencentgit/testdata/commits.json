[
  {
    "id": "e9c5aa72c9e22592ee2611d896b730c1a95cc339",
    "short_id": "e9c5aa72",
    "title": "asd",
    "author_name": "git_user1",
    "author_email": "git_user1@tencent.com",
    "authored_date": "2012-09-19T09:29:19+0000",
    "committed_date": "2012-09-19T09:29:19+0000",
    "committer_name": "git_user1",
    "committer_email": "git_user1@tencent.com",
    "created_at": "2012-09-19T09:29:19+0000",
    "message": "asd"
  },
  {
    "id": "11a63fa022906668338166ce7f2e7bf35d502285",
    "short_id": "11a63fa0",
    "title": "dd",
    "author_name": "git_user1",
    "author_email": "git_user1@tencent.com",
    "authored_date": "2012-09-19T09:29:19+0000",
    "committed_date": "2012-09-19T09:29:19+0000",
    "committer_name": "git_user1",
    "committer_email": "git_user1@tencent.com",
    "created_at": "2012-09-19T09:29:19+0000",
    "message": "dd"
  }
]