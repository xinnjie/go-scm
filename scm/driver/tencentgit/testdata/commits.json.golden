[
  {
    "Sha": "e9c5aa72c9e22592ee2611d896b730c1a95cc339",
    "Message": "asd",
    "Author": {
      "Name": "git_user1",
      "Email": "git_user1@tencent.com",
      "Date": "2012-09-19T09:29:19-00:00",
      "Login": "git_user1",
      "Avatar": ""
    },
    "Committer": {
      "Name": "git_user1",
      "Email": "git_user1@tencent.com",
      "Date": "2012-09-19T09:29:19-00:00",
      "Login": "git_user1",
      "Avatar": ""
    },
    "Link": "https://git.code.tencent.com/xinnjie/testme/commit/e9c5aa72c9e22592ee2611d896b730c1a95cc339"
  },
  {
    "Sha": "11a63fa022906668338166ce7f2e7bf35d502285",
    "Message": "dd",
    "Author": {
      "Name": "git_user1",
      "Email": "git_user1@tencent.com",
      "Date": "2012-09-19T09:29:19-00:00",
      "Login": "git_user1",
      "Avatar": ""
    },
    "Committer": {
      "Name": "git_user1",
      "Email": "git_user1@tencent.com",
      "Date": "2012-09-19T09:29:19-00:00",
      "Login": "git_user1",
      "Avatar": ""
    },
    "Link": "https://git.code.tencent.com/xinnjie/testme/commit/11a63fa022906668338166ce7f2e7bf35d502285"
  }
]